conformance_pack/apigateway.sp

```diff
@@ -63,10 +63,21 @@ control "apigateway_rest_api_stage_use_ssl_certificate" {
     hipaa_final_omnibus_security_rule_2013 = "true"
     nist_800_171_rev_2                     = "true"
     nist_800_53_rev_5                      = "true"
+    nist_csf                               = "true"
     rbi_cyber_security                     = "true"
   })
 }
 
+control "apigateway_rest_api_stage_xray_tracing_enabled" {
```

Comment:
This is a duplicate control please check and remove.

Reply:
This happened while resolving merge conflict.